styles/globals.css

```diff
@@ -1,16 +1,31 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat+Alternates:wght@900&family=Paytone+One&family=Quicksand&display=swap");
```

Comment:
Are there some fonts missing in the import? (Ubuntu, Fira Sans, Droid Sans)

Reply:
I imported these fonts from Google Fonts. Other fonts were already there by default (lines 7,8)